src/config/myanmar/translation.json

```diff
@@ -65,6 +65,18 @@
   "Daytime Land Surface Temperature - 10-day (MODIS)": "សីតុណ្ហភាពផ្ទៃដីពេលថ្ងៃ - 10 ថ្ងៃ (MODIS)",
   "Land Surface Temperature - 10-day Anomaly (MODIS)": "សីតុណ្ហភាពផ្ទៃដី - ភាពមិនប្រក្រតីរយៈពេល 10 ថ្ងៃ (MODIS)",
   "Land Surface Temperature - 10-day Amplitude (MODIS)": "សីតុណ្ហភាពផ្ទៃដី - អំព្លីទីត 10 ថ្ងៃ (MODIS)",
+  "Potential flooding (Landsat-8)": "ទឹកជំនន់ដែលអាចកើតមាន (Landsat-8)",
```

Comment:
@tdlinh2712 just a quick note that this is in Khmer, which is not the language used in Myanmar. It's ok for now for tetsing puposes.

@wadhwamatic are there any plans to get the translation for Myanmar?

Reply:
@ericboucher - no immediate plans for myanmar translation. khmer is ok for testing. we'll remove it before merging

Reply:
can this be resolved now?  Is the plan to just comment out the import of this translation file?

Reply:
@DanielJDufour i think it should be since @ericboucher removed the import of the translation file?